apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -227,24 +227,8 @@
                     "description" : "The consensus protocol versions with respective effective heights",
                     "type" : "object",
                     "additionalProperties" : false,
-                    "required" : [
-                        "9",
-                        "10",
-                        "11"
-                    ],
-                    "properties" : {
-                        "9" : {
-                            "description" : "Minimum height at which protocol is effective",
-                            "type" : "integer",
-                            "minimum" : 0,
-                            "maximum" : 0
-                        },
-                        "10" : {
-                            "description" : "Minimum height at which protocol is effective",
-                            "type" : "integer",
-                            "minimum" : 0
-                        },
-                        "11" : {
+                    "patternProperties" : {
```

Comment:
I find this name to be very generic, to the point where I don't understand at all what it is meant to signify. Maybe something more specific is better, like `minHeights` (just an example, I'm not familiar with this data enough)?

Reply:
Ignore this, it's JSON schema and not our config 🤦‍♂️